packages/api/src/gateways/eth_edge_request.ts

```diff
@@ -0,0 +1,21 @@
+import { request, Method } from "@cryptify/common/src/helpers/ApiRequest";
+import { CreateWalletRequest } from "@cryptify/common/src/requests/create_wallet_request";
+import { Injectable } from "@nestjs/common";
+import { ConfigService } from "@nestjs/config";
+
+@Injectable()
+export class EthEdgeRequest {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    ethEdgeRequest: <T>(method: Method, path: string, body: any) => Promise<any>;
```

Comment:
This can just be called request

```suggestion
    request: <T>(method: Method, path: string, body: any) => Promise<any>;
```